idmapping-rest/src/main/java/org/uniprot/api/idmapping/service/impl/UniProtKBIdService.java

```diff
@@ -135,4 +158,11 @@ protected List<IdMappingStringPair> streamFilterAndSortEntries(
         return super.streamFilterAndSortEntries(
                 streamRequest, mappedIds, kbIdMappingStreamRequest.isIncludeIsoform());
     }
+
+    private boolean addLineage(String fields) {
```

Comment:
method name is not telling intent. Might you want to change it with
isLineageAllowed etc

Reply:
thank you